simple/dense_directed_matrix.go

```diff
@@ -142,11 +200,40 @@ func (g *DirectedDenseGraph) SetEdgeWeight(e graph.Edge) {
 	g.mat.Set(fid, tid, e.Weight())
 }
 
-func (g *DirectedDenseGraph) RemoveEdge(e graph.Edge) {
+// RemoveEdge removes e from the graph, leaving the terminal nodes. If the edge does not exist
+// it is a no-op.
+func (g *DirectedMatrix) RemoveEdge(e graph.Edge) {
 	g.mat.Set(e.From().ID(), e.To().ID(), g.absent)
```

Comment:
No. That should be a no-op according to the interface definition. I'll fix this.